@here/harp-mapview/lib/MapView.ts

```diff
@@ -930,6 +958,20 @@ export class MapView extends THREE.EventDispatcher {
         this.m_backgroundDataSource = new BackgroundDataSource();
         this.addDataSource(this.m_backgroundDataSource);
 
+        if (this.m_enablePolarDataSource) {
+            const styleSetName =
```

Comment:
I think we should still check for the projection here, otherwise you would also add the polar datasource for flat projections

Reply:
Forget this comment. I saw you do this in updatePolarDataSource